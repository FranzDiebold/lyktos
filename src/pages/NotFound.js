import React from 'react';

import { Section, Hero, HeroBody, Container, Title, Subtitle } from 'bloomer';

function NotFound() {
    return (
        <Section className="has-heaven-bg">
            <Hero isSize="medium">
                <HeroBody>
                    <Container className="has-text-centered">
                        <Title isSize="2"><span role="img" aria-label="magnifying glass">üîç</span> 404</Title>
                        <Subtitle isSize="3">Not found...</Subtitle>
                    </Container>
                </HeroBody>
            </Hero>
        </Section>
    );
}

export default NotFound;
