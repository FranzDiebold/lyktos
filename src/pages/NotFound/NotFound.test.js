import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './NotFound';

it('renders not found page correctly', () => {
    const tree = renderer
        .create(<NotFound />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
