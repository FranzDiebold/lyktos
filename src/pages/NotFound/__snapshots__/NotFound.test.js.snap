// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders not found page correctly 1`] = `
<section
  className="hero is-large has-heaven-bg"
>
  <div
    className="hero-body"
  >
    <div
      className="container has-text-centered"
    >
      <h1
        className="title is-2"
      >
        <span
          aria-label="magnifying glass"
          role="img"
        >
          🔍
        </span>
         404
      </h1>
      <h2
        className="subtitle is-3"
      >
        Not found...
      </h2>
    </div>
  </div>
</section>
`;
